op-scim-bridge/chart/op-scim-bridge-mp/templates/deployer-cleanup.yaml

```diff
@@ -7,16 +41,20 @@ metadata:
     app.kubernetes.io/name: {{ .Values.name }}
     app.kubernetes.io/component: cleanup
 spec:
-  schedule: "*/1 * * * *" # once a minute
-  successfulJobsHistoryLimit: 0
+  schedule: "*/10 * * * *" # every 10 minutes
+  successfulJobsHistoryLimit: 1
```

Comment:
Part of the reason why it was set to `0` was due to the output of `kubectl get pods`:

```bash
$ kubectl get pods
NAME                                  READY   STATUS      RESTARTS      AGE
mpdev-cleanup-27637650-5c9md          0/1     Completed   0             94s
mpdev-op-scim-bridge-9c94cbf8-skrdv   1/1     Running     3 (77s ago)   106s
mpdev-redis-master-779d776f5f-7752p   1/1     Running     0             107s
```

Mostly because it's an extra line that doesn't present any useful information to the user/admin/customer - it has run and completed successfully, and in this new revision, you have it set for the job to be deleted upon success (good idea, btw!). Do you think it's still worth having this `Completed` job presented in the `kubectl` output?

Reply:
Ah, thanks for pointing this out Alice. I initially set it to 1 so that I could monitor the logs on failed and successful jobs, but I agree that it's not worth keeping it at 1 if the trade-off is always having a cleanup pod in completed state show up.

I'll revert it back to `0`.

- [ ] Set ConJob `successfulJobsHistoryLimit` back to `0`

Reply:
Much better! :tada:

<img width="1826" alt="image" src="https://user-images.githubusercontent.com/11049609/179838092-4967878b-bf55-45b1-aa74-287bc91256ca.png">